### PR TITLE
[hotfix] OSF Branding was requested for log in page

### PR DESCRIPTION
**Pull request by HarryRybacki**
## Purpose
Add OSF Branding to the OSF page

## Changes
Addition of OSF navigation bar and footer to the `/login/` page

## Side Effects
n/a

Closes-Issue: #2358

### CODE REVIEW

**Comment by HarryRybacki** [comment]
ping @lbanner

**Comment by sloria** [comment]
Minor, but the sign-in panel is rather tight against the navbar. 

osf___sign_in

Also, this same styling should be used for the forgot password page, yes?

**Comment by HarryRybacki** [comment]
Yep, and two factor code. I'll add those. I'm using treebeard css to handle the margin -- it shouldn't be that tight, here is it how it shows up in FF. @caneruguz did I do something weird?

screen shot 2015-03-30 at 4 38 43 pm

**Comment by caneruguz** [comment]
If the css is there it should look the same everywhere however using treebeard.css might be causing some issues depending on if it's loaded on that specific page. You can check that by inspecting the element that's supposed to have the css and checking in chrome dev tools if those css rules are in fact applied to it. 

I might move the helper classes of treebeard into site.css to resolve this issue, in the meantime you can copy the classes you need to site.css

**Comment by sloria** [comment]
I think `treebeard.css` is included on the page, but only by coincidence because `treebeard` is in the `vendor` bundle. 

Better to not use treebeard CSS for this form, anyway, since the two are unrelated.

**Comment by sloria** [comment]
Once this has been applied to the forgot password and 2fa and the CSS is fixed, this is good to merge. :tophat:

**Comment by caneruguz** [comment]
@Steve can we add the helper classes that treebeard uses to site.css or create a new css file for helper classes? I think it has useful things in there that all pages should be able to use. We are essentially using this less file Chelsea made a while ago: https://github.com/overstreetce/css-helper/blob/master/less/helper.less

We can also just use the css version.

**Comment by sloria** [comment]
@caneruguz Sure. Helper classes can be added to site.css.

File: website/static/css/site.css [hunk 1]
```diff
@@ -2450,3 +2450,147 @@ span#profileFullname{
...
+  background-color: #fff;
+}
+.panel-osf > .panel-footer + .panel-collapse > .panel-body {
+  border-bottom-color: #214762;
+}
+
+/* Classes to create margin sizes */
+/* Left Margin Only */
+.m-l-xs {
+margin-left: 5px;
```
Comment (sloria) [inline website/static/css/site.css]: Fix indentation

**Comment by GaryKriebel** [comment]
The spacing on Login is fine.  Need to have the same spacing for Forgot Password and 2FA.
screen shot 2015-04-01 at 12 13 19 pm
screen shot 2015-04-01 at 12 13 31 pm


Can we kill the signin boxes and forgot password link on the nav bar on these three pages?

**Comment by HarryRybacki** [comment]
@GaryKriebel fixed

**Comment by GaryKriebel** [comment]
Spacing looks good.

Is it possible to remove the boxes from the bar on those pages?

**Pull request merged**